core/common/src/LocalTime.kt

```diff
@@ -80,10 +106,13 @@ public expect class LocalTime : Comparable<LocalTime> {
     /** Returns the nanosecond-of-second time component of this time value. */
     public val nanosecond: Int
 
-    /** Returns the time as a second of a day, from 0 to 24 * 60 * 60 - 1. */
+    /** Returns the time as a second of a day, in `0..24 * 60 * 60 - 1`. */
     public fun toSecondOfDay(): Int
 
-    /** Returns the time as a nanosecond of a day, from 0 to 24 * 60 * 60 * 1_000_000_000 - 1. */
+    /** Returns the time as a second of a day, in `0..24 * 60 * 60 * 1_000 - 1`. */
+    public fun toMillisecondOfDay(): Int
+
+    /** Returns the time as a nanosecond of a day, in `0..24 * 60 * 60 * 1_000_000_000 - 1`. */
```

Comment:
Maybe it's worth to put the range end bound in such complex range expressions in parentheses.

Reply:
I took a slightly different approach. WDYT?